### PR TITLE
Update CapitalOne.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#10782

### CODE REVIEW

File: src/chrome/content/rules/CapitalOne.com.xml [hunk 1]
```diff
@@ -27,11 +17,17 @@ Fetch error: http://travelrewardspn.capitalone.com/ => https://travelrewardspn.c
 	- frapplication
 	- frapplication1
 	- frapplication2
+
+	Invalid certificate:
```
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: Just an incomplete cert chain

File: src/chrome/content/rules/CapitalOne.com.xml [hunk 2]
```diff
@@ -52,18 +47,13 @@ Fetch error: http://travelrewardspn.capitalone.com/ => https://travelrewardspn.c
 	<target host="jobs.capitalone.com" />
 	<target host="loanhub.capitalone.com" />
 	<target host="loanhub2.capitalone.com" />
-	<target host="login.capitalone.com" />
-	<target host="login1.capitalone.com" />
-	<target host="login2.capitalone.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: Maybe a valid test url works for them, e.g. `http://login.capitalone.com/resources/images/icon_padlock.gif`

Comment (koops76) [inline src/chrome/content/rules/CapitalOne.com.xml]: @J0WI No, they time out.

File: src/chrome/content/rules/CapitalOne.com.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/CapitalOne.com.xml]: @J0WI Fixed.

File: src/chrome/content/rules/CapitalOne.com.xml (hunk 2 above)
Comment (J0WI) [inline src/chrome/content/rules/CapitalOne.com.xml]: Okay, can you please document this?

Comment (koops76) [inline src/chrome/content/rules/CapitalOne.com.xml]: @J0WI Done.

**Pull request merged**